custom/icds_reports/ucr/tests/test_expressions.py

```diff
@@ -1,36 +1,48 @@
+from copy import deepcopy
 import uuid
 from django.test import TestCase, override_settings
 import mock
 from casexml.apps.case.mock import CaseStructure, CaseFactory
 from casexml.apps.case.tests.util import delete_all_cases, delete_all_xforms
-from corehq.apps.es.fake.forms_fake import FormESFake
 from corehq.apps.userreports.expressions.factory import ExpressionFactory
 from corehq.apps.userreports.specs import EvaluationContext
 from corehq.form_processor.interfaces.dbaccessors import FormAccessors
 
+es_form_cache = []
```

Comment:
Yeah. I really didn't like it either. I wasn't sure on how to mock this without a lot of extra work. I'm happy to refactor if there's a better pattern I can use